### DISCUSSION TITLE
Change symbol type

### DISCUSSION THREAD

**Discussion by JLo5678** [OpenOrienteering/mapper#2054] [Questions & Answers] [answered]
Hi, I’m making a new street-o map.  I open a template which is a osm file and contains man-made features, like benches.  OOM imports them as purple points, so I select them all, then click the man-made feature symbol and right-click to “change all selected objects” to this symbol.  When I do this, it changes them all to a bolder symbol instead.   Am I doing something wrong?  Also, for line types, I can change their colour, but not their thickness.  I appreciate any suggestions.  I haven’t mapped before!
Thanks

**Comment by dl3sdo** [comment, marked as answer]
Hi @JLo5678,
I just created a new map, imported an osm file, selected two of the imported objects being of the same type and changed them to the man-made feature as you described it => it worked as I expected.

What do you mean by changing the colour and thickness of line types?
You do those changes in the respective symbol dialog?

Please tell us your OS and Mapper version, as well as which symbol set and scale you chose. Can you reproduce the issue with a small osm sample that you can attach to the discussion here?

Best Regards, Matthias

**Reply by JLo5678** [reply]
Hi, Thanks for getting back to me.  I'm using Windows 11 and Mapper 0.9.5.  I tried doing as you and starting a fresh map; loaded the osm data, and switched imported features to man-made.  It worked fine.  There must be something corrupted with my project.  When I click on one of these imported features that I switched to man-made in my project, it highlights the correct symbol in the symbol list, but it doesn't look that way on the map.  I attached a screenshot.  I'm not sure why my symbol set is so different from the default.  I'm new to this so I wouldn't have been smart enough to do anything custom.

When I change the line thickness by right clicking the symbol.  It doesn't change in the map.  I super exaggerated it to 0.5 and the line in the map doesn't change (second screenshot).  I also can't get trails to show up as dashed lines.  Heck I just attached my whole project.  I'm not vey far along.
[Brentwood.zip](https://github.com/OpenOrienteering/mapper/files/8338572/Brentwood.zip)
![manmade](https://user-images.githubusercontent.com/102234610/159846161-9cfec0a1-7390-4b00-ad3b-32917cfac5cf.png)
![linethick](https://user-images.githubusercontent.com/102234610/159846236-46761e4a-a06b-4ccf-bc6d-f418aa6ae308.png)

Best Regards,
Jeremy

**Reply by dg0yt** [reply]
> When I change the line thickness by right clicking the symbol. It doesn't change in the map. I super exaggerated it to 0.5 and the line in the map doesn't change (second screenshot). I also can't get trails to show up as dashed lines. Heck I just attached my whole project.

In your map file, "Baseline view" is enabled, along with "Hatch areas". You can toogle these options in the "View" menu.

**Reply by dg0yt** [reply]
PS: This also explains the point symbol appearance...

**Reply by mlerjen** [reply]
Press F2 and F3 to toggle both.

**Reply by JLo5678** [reply]
Thank you all for the help!  I’m making great progress now!

**Comment by dg0yt** [comment]
Summary from https://github.com/OpenOrienteering/mapper/discussions/2054#discussioncomment-2425383:

In your map file, "Baseline view" is enabled, along with "Hatch areas". 
You can toogle these options in the "View" menu, or press `F2` and `F3` to toggle them.
